src/debugger/gdb-stub.c

```diff
@@ -65,7 +65,13 @@ static void _gdbStubEntered(struct mDebugger* debugger, enum mDebuggerEntryReaso
 				// Fall through
```

Comment:
The logic inside of WATCHPOINT_WRITE should probably be removed then. However, gdb will get confused about why it's breaking.